js/addToTask.js

```diff
@@ -34,9 +35,9 @@ function addToDo(taskName, id) {
   const item = `<li class="item">                  
                   <p class="text">${taskName}</p>
                   <div class = "date" id = "${id}"> Created At:  ${dateString} </div>
-                  <img src= "icons/done.svg" data-job = "${COMPLETE}" id ="${id}"/>
-                  <img src = "icons/edit.svg" data-job = "${EDIT}" id = "${id}"/>
-                  <img src = "icons/delete.svg" data-job="${DELETE_TODO}" id="${id}"/>
+                  <img src= "icons/done.svg" data-job = "${COMPLETE}" id ="${id}" style = "inline-block"/>
+                  <img src = "icons/edit.svg" data-job = "${EDIT}" id = "${id}" style = "inline-block"/>
+                  <img src = "icons/delete.svg" data-job="${DELETE_TODO}" id="${id}" style = "inline-block"/>
```

Comment:
incorrect use of spaces around attributes, format correctly.

Reply:
done